accounts/backends.py

```diff
@@ -0,0 +1,46 @@
+from typing import Any, Optional
+
+from django.contrib.auth.backends import ModelBackend
+from django.http import HttpRequest
+
+from accounts.models import User
+
+TWO_FACTOR_AUTH_SESSION_KEY = "using_2FA"
+
+
+class TwoFactorAuthenticationBackend(ModelBackend):
+    """Grabs a regular user, but checks OTP as well as password.
+
+    DEPRECATED
```

Comment:
why is this marked deprecated?

Reply:
oh dear that's not right. Initially it was, but then I started using it again for Admin views